acceptance/README.md

```diff
@@ -0,0 +1,113 @@
+# Acceptance tests for puppetlabs-puppet_agent
+
+## Background
+
+### About Beaker
+
+Beaker is a host provisioning and an acceptance testing framework. If you are
+unfamiliar with beaker, you can start with these documents:
+
+- [The Beaker DSL document](https://github.com/puppetlabs/beaker/blob/master/docs/how_to/the_beaker_dsl.md) will help you understand the test code in the `tests/` and `pre_suite/` subdirectories.
+- [The Beaker Style Guide](https://github.com/puppetlabs/beaker/blob/master/docs/concepts/style_guide.md) will help you write new test code.
+- [Argument Processing](https://github.com/puppetlabs/beaker/blob/master/docs/concepts/argument_processing_and_precedence.md) and [Using Subcommands](https://github.com/puppetlabs/beaker/blob/master/docs/tutorials/subcommands.md) have more information on beaker's command line and environmental options.
+
+### About these tests
+
+Testing this module necessarily involves repeatedly installing and
```

Comment:
I'd add a note that the reason we setup/remove puppet-agent for each test is b/c we're testing stuff related to upgrades.

Reply:
Still would be useful to mention something about upgrades -- I'd prefer not having to think about why it is necessary for us to repeatedly install and uninstall the puppet-agent when it can be explicitly stated.